test/controllers/api/audio_files_controller_test.rb

```diff
@@ -26,6 +27,7 @@
         size: 1024,
         duration: 30
       }
+      @request.env["CONTENT_TYPE"] = "application/json"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.